test/framework/rancher_helpers.go

```diff
@@ -101,6 +107,101 @@ func RancherGetClusterKubeconfig(ctx context.Context, input RancherGetClusterKub
 	result.TempFilePath = tempFile.Name()
 }
 
+// RancherGetOriginalKubeconfig will get the unmodified Kubeconfig for a cluster from Rancher.
+func RancherGetOriginalKubeconfig(ctx context.Context, input RancherGetClusterKubeconfigInput, result *RancherGetClusterKubeconfigResult) {
```

Comment:
Would it make sense to have this and `RancherGetClusterKubeconfig` into a single function? Looks like the differences in the logic are rather small. Maybe we can have an extra field in `RancherGetClusterKubeconfigInput` to set whether we're getting one kubeconfig or the other? Just wondering if this would be viable and so we minimize code repetition.

Reply:
Thanks, that makes sense.

Reply:
Generally speaking the separate kubeconfig is required only for one scenario, when a rancher API server is not available. So it is not required to use the second method, if this failure scenario is not assumed.